src/todo/entities/todo.entity.ts

```diff
@@ -0,0 +1,21 @@
+import { User } from 'src/user/entities/user.entity';
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
+
+@Entity()
+export class Todo {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ length: 255 })
+  title: string;
+
+  @Column({ default: false })
+  isCompleted: boolean;
+
+  @ManyToOne(() => User, (user) => user.todos, {
+    onDelete: 'CASCADE',
+    onUpdate: 'CASCADE',
+    nullable: false,
+  })
+  user: User;
```

Comment:
It seems that not all columns are added to the entity. Where is deadline column?